app/src/main/java/com/example/note/NoteListFragment.java

```diff
@@ -0,0 +1,150 @@
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        setHasOptionsMenu(true);
+        mLayoutView = inflater.inflate(R.layout.note_list_layout, container, false);
+        return mLayoutView;
+    }
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        imageView = mLayoutView.findViewById(R.id.logo);
+        refresh();
+
+
+        noteListView.setOnItemClickListener(new AdapterView.OnItemClickListener(){
+            @Override
+            public void onItemClick(AdapterView<?> arg0, View arg1, int position, long id) {
+                TMANote note = (TMANote)noteList.get(position);
+                Bundle dataBundle = new Bundle();
+                dataBundle.putInt("id", note.getId());
+                Intent intent = new Intent(getActivity().getApplicationContext(), DisplayNoteActivity.class);
+                intent.putExtras(dataBundle);
+                startActivityForResult(intent, REQUEST_CODE_NEW_NOTE);
+            }
+        });
+    }
+
+    public void refresh(){
+        mydb = new TakeMeAwayDBHelper(getActivity());
+
+        noteList = mydb.getNoteList(TakeMeAwayDBHelper.ROW_COUNT_SMALL, TakeMeAwayDBHelper.DESC_ORDER);
+        if(imageView!= null) {
+            if(noteList.size()==0){
+                imageView.setVisibility(View.VISIBLE);
+            } else {
+                imageView.setVisibility(View.GONE);
+            }
+        }
+        ArrayList<String> array_list = new ArrayList<String>();
+
+        for (int i = 0; i < noteList.size(); i++) {
+            TMANote note = (TMANote) noteList.get(i);
+            array_list.add(note.getDate()+" " +note.getTime()+ "     " + note.getTitle());
+        }
+        arrayAdapter = new CustomAdapter(getActivity(), android.R.layout.simple_expandable_list_item_1, array_list);
+
+        noteListView = mLayoutView.findViewById(R.id.note_list);
+        noteListView.setAdapter(arrayAdapter);
+
+        createButton = getActivity().findViewById(R.id.create_new_button);
+
+        createButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Intent intent = new Intent(getActivity().getApplicationContext(), DisplayNoteActivity.class);
+                startActivity(intent);
+            }
+        });
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.search_btn:
+                Toast.makeText(getActivity(),"Hello",Toast.LENGTH_LONG).show();
+                SearchList(item);
+                return true;
+            default:
+                break;
+        }
+
+        return false;
+    }
+
+    public void SearchList(MenuItem item){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.